src/components/modals/WalletListModal.js

```diff
@@ -29,7 +29,8 @@ type OwnProps = {
   showWalletCreators: boolean,
   excludedTokens: Array<string>,
   noWalletCodes: Array<string>,
-  disableZeroBalance: boolean
+  disableZeroBalance: boolean,
+  filter?: string
```

Comment:
@amanalili has a branch which completely refactors the wallet list modal to take much simpler props, like `supportedCurrencyCodes` / `blockedCurrencyCodes`. I'm pretty sure his PR will conflict with this change, so I'm not sure what the correct order of merging & rebasing is here.

Reply:
@swansontec Arts changes actually remove the filter preventing two identical currency codes from being used in the exchange scene. This basically makes @passabilities changes to wallet list modal unnecessary. Lets wait for Art's changes to get merged and this branch can be rebased on top of it.